pkgs/development/ocaml-modules/tty/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildDunePackage
+, fetchurl
+, uutf
+}:
+
+buildDunePackage rec {
+  pname = "tty";
+  version = "0.0.2";
+
+  minimalOCamlVersion = "5.1";
+
+  src = fetchurl {
+    url = "https://github.com/leostera/tty/releases/download/${version}/tty-${version}.tbz";
+    hash = "sha256-DVXkgyOFAHvcY7xlPRWY29jr7qQl4BCfH2lK/6cw8y0=";
```

Comment:
The hash does not seem to be correct.

Reply:
Yep you are right, changing to `sha256-eeD5Y+/QXZzFoEHvOSZj2Q74V8BK5j3Lu3Zsrj2YUUs=` makes it build.

Reply:
Gotcha -- for future reference, how did you compute this hash?

I did so by running:

```
openssl dgst -sha256 -binary ./tty-0.0.1.tbz | openssl base64 -A
```

Reply:
OK -- hash updated!

Reply:
The command looks good. The file name does not. Did you compute the hash of the right file?

Reply:
I usually just leave it empty or set `lib.fakeSha256` which just puts sha256-AAAAAAAAAAAAAAAAAAAAAAAAA then you can build the derivation and it will tell you what the hash is. Sometimes you don't wont to compute the hash by hand because nix fetcher do some stuff to clean up after fetching to make it reproducible.

Reply:
Good to know -- thanks for the tip!